### PR TITLE
Transfer of ownership

### PR DESCRIPTION
**Pull request by pa3ng**
## Proposed change/fix

**UPDATE**: Add transfer of ownership functionality for **MVP v0.2**

Implement `TransferAssertionAction` transaction
   - Remove ingestion org as owner ✅
   - Mark owning entity as the new owner, giving ADMIN and TRANSACTOR roles ✅
   - Delete Assertion from state ✅

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (could be a small readme update, documentation contribution, etc.)

## How to run/test

1. `cargo test`
2. TBD

### CODE REVIEW

File: src/handler/assertion.rs [hunk 1]
```diff
@@ -442,4 +564,237 @@ mod tests {
...
     }
+
+    #[test]
+    /// Test that TransferAssertionAction for a FACTORY assertion is valid.
+    /// The factory assertion should be removed from state.
+    fn test_transfer_assertion_action_for_factory() {
+        let mut transaction_context = MockTransactionContext::default();
+        let mut state = ConsensourceState::new(&mut transaction_context);
+
+        //add ingestion agent
```
Comment (adeebahmed) [inline src/handler/assertion.rs:575]: Does clippy not care about these comments?

Comment (adeebahmed) [reply src/handler/assertion.rs:575]: The format

Comment (pa3ng) [reply src/handler/assertion.rs:575]: apparently not lol @adeebahmed i was following what was already in place

Comment (pa3ng) [reply src/handler/assertion.rs:575]: want me to add the spaces?

Comment (adeebahmed) [reply src/handler/assertion.rs:575]: Weird. Thought it would. Yeah if you can please.

**Review by BuoyLynn — APPROVED** [review]
thanks for keeping it up for us to review. 👀  🧩

**Review by trevormcdonald — APPROVED** [review]

**Review by adeebahmed — APPROVED** [review]

**Pull request merged**